dist/styles/angular-patternfly.css

```diff
@@ -272,3 +272,17 @@
   color: #333;
   margin-left: 10px;
 }
+
+.simple-filter a {
+  cursor: pointer;
+}
+
+.filter-select .btn-default {
+  font-family: 'Open Sans', Helvetica, Arial, sans-serif;
+  font-size: 12px;
+  font-style: italic;
```

Comment:
This might cause alliasing since we don't have a font library defined for italic